packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -23,49 +23,88 @@ import './page.css';
+
+In this example, `<Masthead>` sets the `display` property equal to `"{{ default: 'stack' }}"` to stack the masthead items and also passes in `"inset={{ default: 'insetXs' }}"` to adjust the inset padding.
 
 ```ts file="./PageTertiaryNav.tsx"
 ```
 
-### With or without fill
+### Uncontrolled nav
+
+When the `isManagedSidebar` property is true, it manages the sidebar open/close state, removing the need to pass both `isNavOpen` into the `<PageSidebar>` and `onNavToggle` into the `<PageHeader>`.
+
+```ts file="./PageUncontrolledNav.tsx"
+```
+
+### Filled page sections
+
+To make a `<PageSection>` fill the available vertical space of the page, set the `isFilled` property equal to true.
+
+By default, the last page section sets `isFilled` equal to true.
 
 ```ts file="./PageWithOrWithoutFill.tsx"
 ```
 
 ### Main section padding
 
-```ts file="./PageMainSectionPadding.tsx"
-```
+To adjust the padding of a `<PageSection>`, you can pass in different values to the `padding` property. These value should be aligned to various breakpoints, including 'default', 'sm', 'md', 'lg', 'xl', '2xl'. 
 
-### Uncontrolled nav
+To remove padding, pass 'noPadding' into `padding`. For example, the second section in this example passes in `padding={{ default: 'noPadding' }}`. Since no specific breakpoints are mentioned, every breakpoint will have 'noPadding'.
 
-```ts file="./PageUncontrolledNav.tsx"
-```
+To add padding on specific breakpoints, specify the value for those breakpoints. For example, the third section in this example passes in `padding={{ default: 'noPadding', md: 'padding', lg: 'padding' }}`. On 'md' and 'lg' breakpoints, the default value will be overwritten, and padding will be added.
 
-### Group section
+To remove padding at specific breakpoints, pass in 'noPadding' at those breakpoints. For example, the fourth section in this example passes in `padding={{ md: 'noPadding' }}`, which means that 'md' is the only breakpoint that will remove padding.
 
-```ts file="./PageGroupSection.tsx"
+```ts file="./PageMainSectionPadding.tsx"
 ```
 
-### Vertical nav using PageHeader component
+### Group section
 
-This example is provided because PageHeader and PageHeaderTools are still in use; however, going forward Masthead and Toolbar should be used to make headers rather than PageHeader and PageHeaderTools.
+To group page content sections, add one or more `<PageGroup>` components to a `<Page>`. 
 
-```ts file="./PageVerticalNavUsingPageHeaderComponent.tsx"
+The following example adds a group containing both the `<PageNavigation>` and a section of content.
+
+```ts file="./PageGroupSection.tsx"
 ```
 
 ### Centered section
 
+When a width-limited page section is wider than the value of --pf-c-page--section--m-limit-width--MaxWidth, the section will be centered in the main section.
+
+The content in this example is placed in a card to better illustrate how the section behaves when it is centered. A card is not required to center a page section.
```

Comment:
I'm actually not seeing any difference in this example between having `isWidthLimited` and `isCenterAligned` applied vs removing them. I think a follow up to make the purpose of the props used more clear would be good, unless I'm missing the intent.

We should probably avoid using the same text from the example, though, and just mention what props to set to center the PageSection

Reply:
@mcoker WDYT?

Reply:
Here's the difference:

Default:
<img width="2609" alt="Screenshot 2022-11-18 at 11 56 21 AM" src="https://user-images.githubusercontent.com/35148959/202771355-a1280473-eb6a-4257-babe-cef844d62609.png">

`isWidthLimited`:
<img width="2609" alt="Screenshot 2022-11-18 at 11 56 42 AM" src="https://user-images.githubusercontent.com/35148959/202771382-2d6c4f1f-7864-445d-bdf1-92d9816fdb7a.png">

`isCenterAligned`:
<img width="2609" alt="Screenshot 2022-11-18 at 11 56 49 AM" src="https://user-images.githubusercontent.com/35148959/202771423-51395e61-c291-407e-b3a2-c550bc8a11c6.png">

Core has an example that shows a center aligned section with (first section) and without (second, card gallery) a card wrapping the content to shows the boundary. We also have a third section that I don't know is needed, but shows how you can center the text, though it uses a utility class that isn't tied to the page component CSS at all.

https://www.patternfly.org/v4/components/page/html-demos/centered-section/